src/ui/profile_preferences.ui

```diff
@@ -2665,6 +2665,20 @@ you can use it but there could be issues with aligning columns of text</string>
             </property>
            </widget>
           </item>
+          <item row="1" column="2">
+           <widget class="QLabel" name="label_6">
```

Comment:
Please give this label a more meaningful name than just `label_6` the default of `label_` and then the next unused positive integer is not helpful if hand editing/reviewing this type of file.
```suggestion
           <widget class="QLabel" name="label_mapInfoBg">
```

Reply:
Also make this label be a "buddy" of the `pushButton_mapInfoBg`  - select the "Mapper colors" tab first then click the "buddy" tab in the editor (see the green circle) then go to the label (which will light up in red as it does not have a buddy) and click and drag the line that forms to the control that it is the buddy of {in the example I show there is not a buddy defined yet for that label). A buddy means that if you select a label (especially by keyboard but also I think by mouse then the focus jumps to the control that the label is the buddy of)....
![image](https://user-images.githubusercontent.com/6163092/156080422-d7bbba95-1ba5-41a1-a3c3-7e28a4bf03bb.png)

To go back to normal "Widget" mode afterwards (or if you want to change the tab you are working on in the editor!) click the tab two to the left of the "Buddy" mode one.